modules/ui/app/scripts/controllers/ApplicationsCtrl.js

```diff
@@ -60,7 +60,7 @@ angular.module('wasabi.controllers').
             $scope.loadApplications();
 
             $scope.openApplicationModal = function (application) {
-                var modalInstance = $modal.open({
+                var modalInstance = $uibModal.open({
```

Comment:
I guess I would say no.  The class name and the associated "$ var" was changed to make the global name space a little less likely to have conflicts (I think) and to reflect that these things are related to the Bootstrap UI (hence "uib").  But the local var is still a reference to the "modal dialog instance".